terraform/projects/app-ecs-instances/instance-user-data.tpl

```diff
@@ -32,10 +32,22 @@ else
   echo "[$(date '+%H:%M:%S %d-%m-%Y')] attach-volume: /dev/$DEVICE is already formatted: $(file -s /dev/$DEVICE)"
 fi
 
-# Allow prometheus container user access to read/write/execute within container
+# Allow prometheus container user access to read/write metrics to persistent volume mount from within a container
+
+#Mount volume to be used by prometheus container
 mkdir -p /ecs/prometheus_data
 mount /dev/$DEVICE /ecs/prometheus_data
-chmod 777 /ecs/prometheus_data
+
+
+
+#Create prometheus group and allow it to read and write to our volume for storing prometheus data. Note, 65534 is 
+#chosen as the UID to be added to the prometheus group as this is the UID that prometheus in the docker container runs as.
+
+#Create prometheus & grant access to prometheus data files. We create the group & match it to the gid
```

Comment:
I don't think we need this line 46 anymore as 43 and 44 are fairly descriptive and feel like they are already saying this. Do you agree?

Reply:
Yep, I agree